changelogs/unreleased/global-names.md

```diff
@@ -4,3 +4,8 @@
   the entire cluster with all its replicasets. It has to match in all instances
   of all the replicasets in one cluster and is displayed in
   `box.info.cluster.name` (gh-5029).
+
+* New option `box.cfg.replicaset_name` allows to assign a human-readable name to
+  the replicaset. Works the same as `box.cfg.replicaset_uuid`. It has to match
+  in all instances of one replicaset and is displayed in
+  `box.info.replicaset.name` (gh-5029).
```

Comment:
I didn't hear about this idea anything until you mentioned it here. At the same time I don't see why would it even be needed.

Reply:
> I heard an idea about generating instance/replicaset UUID based on the instance/replicaset name (hashing alike). I guess, from @Mons. Is it declined? If so, why?

I think you're mentioning @ochaton's script which turns a readable string into a uuid (like l33tsp34k but with chars allowed for a uuid). It seems this was merely a workaround, and isn't needed anymore since we have normal instance names.

Reply:
Well, ok. Just noted.